src/wp-includes/rest-api/fields/class-wp-rest-meta-fields.php

```diff
@@ -478,7 +478,7 @@ protected function get_registered_fields() {
 
 			$default_schema = array(
 				'type'        => $default_args['type'],
-				'label'       => empty( $args['label'] ) ? '' : $args['label'],
+				'title'       => empty( $args['label'] ) ? '' : $args['label'],
```

Comment:
Woudl it make sense to call it `title` everywhere for consistency?

Reply:
I also considered it when working on the `register_setting` update. The title is specific to JSON Schema.

WP seems to use the `label` property for human-readable labels both in PHP and JS.

Reply:
I thought the same and I followed the same reasoning. It seems `label` is used in other functions:
* [register_post_type](https://developer.wordpress.org/reference/functions/register_post_type/#parameters).
* [register_taxonomy](https://developer.wordpress.org/reference/functions/register_taxonomy/#parameters).
* [register_setting](https://developer.wordpress.org/reference/functions/register_setting/#parameters).

Having said that, both options are fine to me.

Reply:
I’m fine with `label` if there are multiple instances of it in so impactful APIs.